docs/content/docs/recipes/temporary-storage.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Using Fluid for temporary storage
+menuPosition: 3
+---
+Some developers may wish to use the Fluid container for temporary storage only, choosing to use another data store for the long-term storage of the data after the collaboration period has passed – or not storing the data at all, as it may have no usefulness beyond the collaboration period.
+
+In this scenario, we assume the data in the primary data store will be temporarily copied to a Fluid container so that it can participate in a real-time collaboration experience for some period of time, and that it will not be edited in the original location while it’s in the Fluid container. All of the business logic described here is assumed to be executed on the client in the context of the current user.
+
+![high level diagram of data import and export](https://fluidframework.blob.core.windows.net/static/images/import-export.jpg)
+
+## Getting data into the container
+
+The application developer is responsible for connecting to the external data store from their client-side web application, finding the appropriate data, and marshalling the data into a new Fluid container. We recommend creating the container in a detached state, inserting the external data into new Fluid objects, and then attaching the container - to avoid the situation where another user opens the container and the seed data is not yet present. For more information on using detached containers, see {working with detached containers page}.
+
+## Locking the source data
+
+Keeping data in sync between a Fluid container and an external data store is beyond the scope of this discussion. If the contents of the Fluid container are going to be written back to the original data store, the developer should take some precaution to ensure that users aren’t writing to the original data store while the data is also being edited in the Fluid container.
+
+## Data permissions
+
+The user that initiates the transfer of the data into the Fluid container would need to have permissions to read the source data and set whatever locking mechanism is used to keep other users from editing the source data while it resides in the Fluid container. The other users in the collaboration experience do not need to have access to the original data store – this enables developers to delegate editing of specific data elements. Because the data transfer is all managed on the client, only users with access to the original data store should have the option to save changes back to the source. The developer is responsible for managing the user experience of the “save” action, including which clients have the option to save updates back to the source of truth.
```

Comment:
```suggestion
The user that initiates the transfer of the data into the Fluid container would need to have permissions to read the source data and set whatever locking mechanism is used to keep other users from editing the source data while it resides in the Fluid container. The other users in the collaboration experience do not need to have access to the original data store. This enables developers to delegate editing of specific data elements. Because the data transfer is all managed on the client, only users with access to the original data store should have the option to save changes back to the source. The developer is responsible for managing the user experience of the "save" action, including which clients have the option to save updates back to the source of truth.
```